tests/DatastreamTest.php

```diff
@@ -244,7 +261,115 @@ public function testContentXFromFile() {
     $this->assertEquals('<testFixture></testFixture>', trim($newds->content));
   }
 
-  public function testContentXFromUrl() {
+  public function testContentXFromUrlHttpLoc() {
+    $data = <<<foo
```

Comment:
Not sure this is actually specified in coding standards anywhere, but... seems unusual to see lower-case identifier used for a heredoc. (though I now see that it was [used here previously](https://github.com/Islandora/tuque/pull/165/files#diff-31c23062c8a28edf3fc51adb10834190R373))

... Actually, on second thought, should this XML be moved into [a static fixture](https://github.com/Islandora/tuque/pull/165/files#diff-31c23062c8a28edf3fc51adb10834190R258), since [it's used twice](https://github.com/Islandora/tuque/pull/165/files#diff-31c23062c8a28edf3fc51adb10834190R319)? (might give the one the same treatment?)